packages/integration-tests/projects/suite-web/tests/wallet/check-xpub-ada.test.ts

```diff
@@ -0,0 +1,52 @@
+// @group:suite
+// @retry=2
+
+describe('Check Cardano XPUB', () => {
+    beforeEach(() => {
+        cy.task('startEmu', { wipe: true });
+        cy.task('setupEmu', {
+            needs_backup: false,
+            mnemonic: 'all all all all all all all all all all all all',
+        });
+        cy.task('startBridge');
+
+        cy.viewport(1080, 1440).resetDb();
+        cy.prefixedVisit('/settings/coins');
+        cy.passThroughInitialRun();
+    });
+
+    afterEach(() => {
+        cy.task('stopEmu');
+    });
+
+    /* Test case
+    1. Pass onboarding
+    2. Disable BTC coin to speedup discovery process
+    3. Enable Ada coin
+    4. Navigate to Accounts/ADA/Account/
+    5. Click on @wallets/details/show-xpub-button
+    6. Check that @xpub-modal/xpub-field is present
+    */
+    it.only('Check Cardano XPUB', () => {
```

Comment:
you most likely don't want to leave `only` here.

Reply:
@mroz22 done. sorry i will check it next time ;)